From patchwork Fri May 28 14:07:36 2021
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Alex Bee <knaerzche@gmail.com>
X-Patchwork-Id: 12287077
Return-Path: 
 <SRS0=NyFe=KX=lists.infradead.org=linux-rockchip-bounces+linux-rockchip=archiver.kernel.org@kernel.org>
X-Spam-Checker-Version: SpamAssassin 3.4.0 (2014-02-07) on
	aws-us-west-2-korg-lkml-1.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-13.7 required=3.0 tests=BAYES_00,
	DKIM_ADSP_CUSTOM_MED,DKIM_SIGNED,DKIM_VALID,FREEMAIL_FORGED_FROMDOMAIN,
	FREEMAIL_FROM,HEADER_FROM_DIFFERENT_DOMAINS,INCLUDES_CR_TRAILER,
	INCLUDES_PATCH,MAILING_LIST_MULTI,SPF_HELO_NONE,SPF_PASS,USER_AGENT_GIT
	autolearn=ham autolearn_force=no version=3.4.0
Received: from mail.kernel.org (mail.kernel.org [198.145.29.99])
	by smtp.lore.kernel.org (Postfix) with ESMTP id 36E53C4708C
	for <linux-rockchip@archiver.kernel.org>;
 Fri, 28 May 2021 14:08:54 +0000 (UTC)
Received: from bombadil.infradead.org (bombadil.infradead.org
 [198.137.202.133])
	(using TLSv1.2 with cipher ECDHE-RSA-AES256-GCM-SHA384 (256/256 bits))
	(No client certificate requested)
	by mail.kernel.org (Postfix) with ESMTPS id EA2FB613B6
	for <linux-rockchip@archiver.kernel.org>;
 Fri, 28 May 2021 14:08:53 +0000 (UTC)
DMARC-Filter: OpenDMARC Filter v1.3.2 mail.kernel.org EA2FB613B6
Authentication-Results: mail.kernel.org;
 dmarc=fail (p=none dis=none) header.from=gmail.com
Authentication-Results: mail.kernel.org;
 spf=none
 smtp.mailfrom=linux-rockchip-bounces+linux-rockchip=archiver.kernel.org@lists.infradead.org
DKIM-Signature: v=1; a=rsa-sha256; q=dns/txt; c=relaxed/relaxed;
	d=lists.infradead.org; s=bombadil.20210309; h=Sender:
	Content-Transfer-Encoding:Content-Type:List-Subscribe:List-Help:List-Post:
	List-Archive:List-Unsubscribe:List-Id:MIME-Version:Message-Id:Date:Subject:Cc
	:To:From:Reply-To:Content-ID:Content-Description:Resent-Date:Resent-From:
	Resent-Sender:Resent-To:Resent-Cc:Resent-Message-ID:In-Reply-To:References:
	List-Owner; bh=pWey57zxdoIVbRAVJjqLYNTpBlEvkCFT3N626KtxH6U=; b=UacJDmmYXKk/jo
	/DbyygSLRop3zDIkOFeSMVg8iE/feidw/1OCt7sOZ0Lf+hpw0fEdKFRsx+oau4NO0/dmhGmCrGxiI
	WBM+7QVx+iLzAcqqSvfvFhhSOLB+LHZCFcHOWOuJG5tnX0sxaCIWWWTO7hPcwgfjOCVkhT+Tba2Hm
	bXK2wNe6hUqQequhSk2/VNDM1mpS4pILjACVpbeNOXSA5D4f48z3AxVf4ZuDO8StD4P+z62eVfw2o
	9r35Q+oZ5wc8iOwxwF1ERe4SAIA0I8QwhL/D5xeiSd20ix5o3g2km1YJKn3h08KsSLX51IBmWyYHx
	fHukRA/UM8vsZa5SYhZw==;
Received: from localhost ([::1] helo=bombadil.infradead.org)
	by bombadil.infradead.org with esmtp (Exim 4.94.2 #2 (Red Hat Linux))
	id 1lmdAP-00G2FS-DN; Fri, 28 May 2021 14:08:49 +0000
Received: from mail-ed1-x530.google.com ([2a00:1450:4864:20::530])
 by bombadil.infradead.org with esmtps (Exim 4.94.2 #2 (Red Hat Linux))
 id 1lmd9L-00G1j0-FW; Fri, 28 May 2021 14:07:44 +0000
Received: by mail-ed1-x530.google.com with SMTP id r23so5029901edw.1;
 Fri, 28 May 2021 07:07:42 -0700 (PDT)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=gmail.com; s=20161025;
 h=from:to:cc:subject:date:message-id:mime-version
 :content-transfer-encoding;
 bh=gmeSm8o0KnklsZ4e5IrRfjO+jG/+LdBVNPQF2l/4OlA=;
 b=i0NGwOcoQW/Xe5YSw61VwzlX5fLiNTLQQfXFRqC9saqhlVpRBGI1F/TvMT6xqu74xp
 v0nZOa6qca4TPgUsSucOi2U35jbKRPyMXUAgHEQcZWokuANdlou6lUtXQpn4gqvpIn2/
 YVaKD6DP0Z/G5XcIt1ZpinVB88UexKQUcph/+mPmPqmCjxYow1Er9CV5zEgsHqGVsg6o
 y6PLOMKPOtDQp8IAhBo4MWo6QoKT2PdDsMpPGcYguEhKyw3CZ2XWvAVp+hqDdsLkQ4XU
 4So1qcnp2rFbg1Xw8mv0vR/jDG2HHa2g2qCjhXqCVszFKaHGOqXwXY8qgt9Fr8uioFAI
 U9qg==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
 d=1e100.net; s=20161025;
 h=x-gm-message-state:from:to:cc:subject:date:message-id:mime-version
 :content-transfer-encoding;
 bh=gmeSm8o0KnklsZ4e5IrRfjO+jG/+LdBVNPQF2l/4OlA=;
 b=Sbp3Z+xeH/Kc3kJdRQwE40NcSiJw6NsMQrgodxW8YFwO28EZFbEZP78ZNw9SWkKfaa
 gNc0NYvMu6S247DvTvlc1Esg7hvDx5QDz7CN8B/D81iUYkGE6gFGqvpu9LweYfb8GxoN
 fBTtSJe8KtCPVTA64h1eMCq3ZM1sy3QAGI2cUKi3by8uy0AzcJiI+L2tiHQvYtR2ZyUV
 2CqzgbCFJTQQtdM/8XMMzAWes4DoZQm3to6iUdM6MBzC0yfVNAu51CEusAshrEpTlXCW
 bIXePkiFEHdFPYGsvymksN/XjDbKx5uAqxgxyY2sfa85L79mNHQgughaJ3Sqbrxg/2ab
 6IHg==
X-Gm-Message-State: AOAM5313tCyAXW+UDWl1/mEMyKUv90tjhtYOInK9bD5gCXEp6nYSFpRK
 dgvPq0FNFYMeG0fD6PzX8A==
X-Google-Smtp-Source: 
 ABdhPJxKPgSY9GkyhdHngZk+RvVE3sVtTWqSwT9kL7gKgmXpjd77U+2R98FBLEQ1l0t6+X8VG8yd9g==
X-Received: by 2002:a05:6402:14c1:: with SMTP id
 f1mr9753808edx.334.1622210861554;
 Fri, 28 May 2021 07:07:41 -0700 (PDT)
Received: from localhost.localdomain (ip4d17b4b8.dynamic.kabel-deutschland.de.
 [77.23.180.184])
 by smtp.googlemail.com with ESMTPSA id bh2sm2389309ejb.80.2021.05.28.07.07.40
 (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
 Fri, 28 May 2021 07:07:41 -0700 (PDT)
From: Alex Bee <knaerzche@gmail.com>
To: Heiko Stuebner <heiko@sntech.de>, linux-clk@vger.kernel.org,
 linux-rockchip@lists.infradead.org
Cc: Michael Turquette <mturquette@baylibre.com>,
 Stephen Boyd <sboyd@kernel.org>, linux-arm-kernel@lists.infradead.org,
 linux-kernel@vger.kernel.org, Alex Bee <knaerzche@gmail.com>
Subject: [PATCH] clk: rockchip: export ACLK_VCODEC for RK3036
Date: Fri, 28 May 2021 16:07:36 +0200
Message-Id: <20210528140736.79686-1-knaerzche@gmail.com>
X-Mailer: git-send-email 2.27.0
MIME-Version: 1.0
X-CRM114-Version: 20100106-BlameMichelson ( TRE 0.8.0 (BSD) ) MR-646709E3 
X-CRM114-CacheID: sfid-20210528_070743_586077_073946FC 
X-CRM114-Status: GOOD (  13.67  )
X-BeenThere: linux-rockchip@lists.infradead.org
X-Mailman-Version: 2.1.34
Precedence: list
List-Id: Upstream kernel work for Rockchip platforms
 <linux-rockchip.lists.infradead.org>
List-Unsubscribe: <http://lists.infradead.org/mailman/options/linux-rockchip>,
 <mailto:linux-rockchip-request@lists.infradead.org?subject=unsubscribe>
List-Archive: <http://lists.infradead.org/pipermail/linux-rockchip/>
List-Post: <mailto:linux-rockchip@lists.infradead.org>
List-Help: <mailto:linux-rockchip-request@lists.infradead.org?subject=help>
List-Subscribe: <http://lists.infradead.org/mailman/listinfo/linux-rockchip>,
 <mailto:linux-rockchip-request@lists.infradead.org?subject=subscribe>
Sender: "Linux-rockchip" <linux-rockchip-bounces@lists.infradead.org>
Errors-To: 
 linux-rockchip-bounces+linux-rockchip=archiver.kernel.org@lists.infradead.org

It is required for the series at [1] to let hantro driver aquire the
clock and set the rate for RK3036 correctly, but I didn't want to
add a patch for yet another subsystem to this series.

[1] https://lore.kernel.org/linux-media/20210525152225.154302-1-knaerzche@gmail.com/

Signed-off-by: Alex Bee <knaerzche@gmail.com>
---
 drivers/clk/rockchip/clk-rk3036.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/clk/rockchip/clk-rk3036.c b/drivers/clk/rockchip/clk-rk3036.c
index 91d56ad45817..614845cc5b4a 100644
--- a/drivers/clk/rockchip/clk-rk3036.c
+++ b/drivers/clk/rockchip/clk-rk3036.c
@@ -259,7 +259,7 @@ static struct rockchip_clk_branch rk3036_clk_branches[] __initdata = {
 			RK2928_CLKGATE_CON(1), 13, GFLAGS,
 			&rk3036_uart2_fracmux),
 
-	COMPOSITE(0, "aclk_vcodec", mux_pll_src_3plls_p, 0,
+	COMPOSITE(ACLK_VCODEC, "aclk_vcodec", mux_pll_src_3plls_p, 0,
 			RK2928_CLKSEL_CON(32), 14, 2, MFLAGS, 8, 5, DFLAGS,
 			RK2928_CLKGATE_CON(3), 11, GFLAGS),
 	FACTOR_GATE(HCLK_VCODEC, "hclk_vcodec", "aclk_vcodec", 0, 1, 4,
